salt/metalk8s/internal/sreport/collected.sls

```diff
@@ -0,0 +1,3 @@
+Collect reports:
+  cmd.run:
+    - name: sosreport -o metalk8s -o containerd --batch -kmetalk8s.podlogs -kmetalk8s.describe -kcontainerd.logs -kcontainerd.all
```

Comment:
Suggest this one to include and require the `installed` state and btw remove the `installed` from `init.sls`